<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAMSControl</class>
 <widget class="QWidget" name="FAMSControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDoubleSpinBox" name="kSelect">
     <property name="toolTip">
      <string>Adjust parameter k for adaptive bandwidths</string>
     </property>
     <property name="prefix">
      <string>k = </string>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>4.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pruneButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="runButton">
     <property name="text">
      <string>run</string>
     </property>
     <property name="icon">
      <iconset theme="run-build">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="stopButton">
     <property name="toolTip">
      <string>Stop current computation</string>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
     <property name="icon">
      <iconset theme="process-stop">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPruneClusters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/icons-custom/index.qrc">
     <normaloff>:/icons/prune-clusters.svg</normaloff>:/icons/prune-clusters.svg</iconset>
   </property>
   <property name="text">
    <string>prune small clusters</string>
   </property>
   <property name="toolTip">
    <string>Prune clusters containing less than 0.5% of all proteins</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/icons-custom/index.qrc"/>
 </resources>
 <connections/>
</ui>
